spyder/utils/conda.py

```diff
@@ -230,7 +248,13 @@ def get_spyder_conda_channel():
 
 @lru_cache(maxsize=1)
 def conda_version(conda_executable=None):
-    """Get the conda version if available."""
+    """
+    Get the conda version if available.
+
+    Note: This function can potentialy get the version of other
+    conda-like executables like mamba, micromamba or Pixi, as well as any
+    executable that provides a `--version` CLI argument.
```

Comment:
```suggestion
    Note: This function can get the version of other conda-like executables
    like mamba, micromamba or pixi, as well as any executable that provides a
    `--version` CLI argument.
```

Improve docstring a bit because we're already using this function to get the versions of other conda-like executables (e.g. when `finda_conda` finds micromamba instead of conda, this functions returns the micromamba version in our kernel spec).